<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_services</class>
 <widget class="QDialog" name="Dialog_services">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сервисные службы</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableSevices">
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>№</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Название сервисной службы</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btServicesAdd">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Добавить строку ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btServicesUpdate">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Обновить строку ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btServicesDel">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Удалить строку ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btServicdesFind">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Поиск ...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
